Removed `options` param from lint function

radum opened the pull request:
After this code [commit](https://github.com/yeoman/generator-webapp/commit/796efb90a442f390fa689a776c5f7bab527af537) we no longer need the `options` param when calling the `lint` function.

eddiemonge commented:
probably easier to just change this line: https://github.com/yeoman/generator-webapp/blob/master/app/index.js#L288

neilhem commented:
@eddiemonge I think when generator scaffolds app it creates fonts in inside app folder, but references in Gruntfile.js are suppose to be wrong.

silvenon commented:
I guess it works either way. People like to put them in the `styles` directory because fonts are imported from CSS, but it makes no difference.

neilhem commented:
So when generator scaffolds app it should do it one way create folder in app/fonts & references in Gruntfile.js should be app/fonts too.

silvenon commented:
Yep, that works for me. I prefer to have it next to `styles`, rather than in it. Because you may have other directories within `styles` which contain actual stylesheets, and having `fonts` be one of them is weird.

Thanks for the PR!

radum committed:
Removed `options` param from lint function

After this code [commit](https://github.com/yeoman/generator-webapp/commit/796efb90a442f390fa689a776c5f7bab527af537) we no longer need the `options` param when calling the `lint` function.

silvenon commented:
That is correct, thanks! 👍
